Sources/HummingbirdLambda/Lambda.swift

```diff
@@ -46,4 +56,16 @@ extension HBLambda {
     public static func main() throws {
         HBLambdaHandler<Self>.main()
     }
+
+    public var applicationContext: HBApplicationContext {
+        HBApplicationContext(
+            threadPool: NIOSingletons.posixBlockingThreadPool,
+            configuration: HBApplicationConfiguration(),
+            logger: Logger(label: "hb-lambda"),
+            encoder: JSONEncoder(),
```

Comment:
Can we make the assumption that bodies will be encoded in JSON?

Reply:
Not always, you're right.